codespell_lib/data/dictionary.txt

```diff
@@ -35803,7 +35803,9 @@ visuallisation->visualisation
 visuallization->visualization
 visualy->visually
 visualyse->visualise, visualize,
-visualzation->visualization
+visualzation->visualisation, visualization,
+visulization->visualisation, visualization,
+visulizations->visualisations, visualizations,
```

Comment:
Sorry, I meant the equivalent typo with an s, e.g. `visualsation->visualisation`.

It's unlikely if someone typed ...z... that they actually meant ...s... (unless they didn't know which one to use, in which case offering both will still leave them confused. But e.g. visualiation could usefully offer both variants.

Also you can edit your own files directly if you want without bothering with suggestions, go to Files changed, then ... menu on a file, then Edit file.

Reply:
Thanks for your comment. Are you fine with this suggestion?
```suggestion
visualiation->visualisation, visualization,
visualsation->visualisation,
visualsation->visualisation,
visulisation->visualisation,
visulisations->visualisations,
visulization->visualization,
visulizations->visualizations,
```

Reply:
@tkoyama010 single results do not need a trailing `,`